source/manual/removing-a-user-from-puppet.html.md

```diff
@@ -0,0 +1,26 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Removing a user from Puppet
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+last_reviewed_on: 2018-05-18
+review_in: 12 months
+---
+
+Removing a user from our infrastructure via Puppet is a 2 change process that
+requires a deploy in the middle. The first change ensures that when puppet
+runs the users home directory is removed, the second change removes the
+user from puppet itself. If the user is just removed from puppet their files
+will remain on our servers forever more.
+
+1. First find the user manifest in: [modules/users/manifests][manifest-path]
+1. Add an entry to the govuk_user class of `ensure => absent,` -
+   [example][absent-example]
+1. Once this has been raised as a PR and merged, deploy puppet to all
+   environments
+1. Create another PR that removes the user manifest file
+1. Deploy puppet again
+
+[manifest-path]: https://github.com/alphagov/govuk-puppet/tree/master/modules/users/manifests
+[absent-example]: https://github.com/alphagov/govuk-puppet/pull/7319/files#diff-23436d1831fbc44a8f2bb652b9233d6cR4
```

Comment:
I'd suggest a different example, as the change in this example wouldn't have been very effective, as I believe you need to keep the hieradata when you add `ensure => absent`. Perhaps use  https://github.com/alphagov/govuk-puppet/commit/0757bad41ed577f15c7f5d9e508f55e78c612ddb ?

Reply:
Oh really - I didn't realise it worked like that. I'll update.